projects/packages/jetpack-mu-wpcom/src/features/wpcom-whats-new/wpcom-whats-new.php

```diff
@@ -1,96 +0,0 @@
-<?php
-/**
- * WPCom Whats New support for WordPress.com sites.
- *
- * @package automattic/jetpack-mu-wpcom
- */
-
-/**
- * Check if the WPCom Whats New should be loaded.
- *
- * @return bool
- */
-function should_load_wpcom_whats_new_app() {
-	// Only load on the WPcom platform.
-	if ( ! class_exists( 'Automattic\Jetpack\Status\Host' ) ) {
-		return false;
-	}
-
-	global $pagenow;
-	$allowed_pages = array(
-		'post.php',
-		'post-new.php',
-		'site-editor.php',
-	);
-	return isset( $pagenow ) && in_array( $pagenow, $allowed_pages, true );
-}
-
-/**
- * Enqueue block editor assets.
- */
-function wpcom_whats_new_enqueue_script_and_style() {
-	if ( ! should_load_wpcom_whats_new_app() ) {
-		return;
-	}
-
-	$whats_new_handle = 'wpcom-whats-new';
-	$version          = gmdate( 'Ymd' );
-
-	$data = wp_json_encode(
-		array(
-			'siteId' => get_current_blog_id(),
-		)
-	);
-
-	wp_enqueue_script(
-		$whats_new_handle,
-		'//widgets.wp.com/whats-new/build.min.js',
```

Comment:
Let's also track usage of the app and consider removing it altogether after a few months.

Reply:
Quite a bit of code we can remove 🙂